test/functional/wallet_send.py

```diff
@@ -544,16 +543,9 @@ def run_test(self):
                 break
         psbt_in = dec["inputs"][input_idx]
         # Calculate the input weight
```

Comment:
nit: not sure if this comment is redundant now since the function name is `calculate_input_weight`